lib/api/calendars.ts

```diff
@@ -42,3 +42,10 @@ export const getInbox = (client: DAVClient) => async (userId: string): Promise<C
     ),
   }));
 }
+
+export const deleteEvent = (client: DAVClient) => async (eventPath: string): Promise<string> => {
+  return await client.requestText({
```

Comment:
```suggestion
  return await client.request({
```
no body was sent within the request, so no need to use `requestJson` or `requestText`

Reply:
i must added it to the DAV client